.gitlab-ci.yml

```diff
@@ -49,10 +49,11 @@ include:
     - ci/install-and-build.yml
     - ci/prebuild.yml
     - ci/packages/components.yml
-    - ci/packages/suite-misc.yml
     - ci/packages/suite-web.yml
     - ci/packages/suite-desktop.yml
 #    - ci/packages/suite-native.yml
     - ci/packages/rollout.yml
     - ci/packages/landing-page.yml
     - ci/packages/suite-web-landing.yml
+    # to be run only on scheduled pipelines
+    - ci/schedules.yml
```

Comment:
It's common to add a newline at the end of a file :P.

Reply:
:exploding_head: ok